src/NzbDrone.Core/MetadataSource/SkyHook/SkyHookProxy.cs

```diff
@@ -205,7 +210,92 @@ public Movie GetMovieInfo(int TmdbId, Profile profile = null)
+                        htmlCode = htmlCode.Replace(" ", String.Empty);
+                        cookIdent = "identifier=" + getBetween(htmlCode, "document.cookie=\"identifier=", "\"+expires+\";path=/;domain=.allflicks.net\"");
+                    }
+                    int length = 100;
+                    int start = 0;
+                    string titleForNetflix = movie.Title;
+                    int numFound = 1;
+                    while (start < numFound && !(movie.Year > now_year))
+                    {
+                        string postData = "draw=4&columns[0][data]=box_art&columns[0][name]=&columns[0][searchable]=true&columns[0][orderable]=false&columns[0][search][value]=&columns[0][search][regex]=false&columns[1][data]=title&columns[1][name]=&columns[1][searchable]=true&columns[1][orderable]=true&columns[1][search][value]=&columns[1][search][regex]=false&columns[2][data]=year&columns[2][name]=&columns[2][searchable]=true&columns[2][orderable]=true&columns[2][search][value]=&columns[2][search][regex]=false&columns[3][data]=genre&columns[3][name]=&columns[3][searchable]=true&columns[3][orderable]=true&columns[3][search][value]=&columns[3][search][regex]=false&columns[4][data]=rating&columns[4][name]=&columns[4][searchable]=true&columns[4][orderable]=true&columns[4][search][value]=&columns[4][search][regex]=false&columns[5][data]=available&columns[5][name]=&columns[5][searchable]=true&columns[5][orderable]=true&columns[5][search][value]=&columns[5][search][regex]=false&columns[6][data]=director&columns[6][name]=&columns[6][searchable]=true&columns[6][orderable]=true&columns[6][search][value]=&columns[6][search][regex]=false&columns[7][data]=cast&columns[7][name]=&columns[7][searchable]=true&columns[7][orderable]=true&columns[7][search][value]=&columns[7][search][regex]=false&order[0][column]=5&order[0][dir]=desc&start=" + start.ToString() + "&length=" + length.ToString() + "&search[value]=" + titleForNetflix + "&search[regex]=false&movies=true&shows=false&documentaries=true&rating=netflix&min=1900&max=" + now_year.ToString();
+                        byte[] byteArray = Encoding.UTF8.GetBytes(postData);
+
+                        HttpWebRequest rquest = (HttpWebRequest)WebRequest.Create(url);
+                        rquest.Method = "POST";
+                        rquest.Headers.Add("X-Requested-With", "XMLHttpRequest");
+                        rquest.Headers.Add("Cookie", cookIdent);
+                        rquest.ContentType = "application/x-www-form-urlencoded; charset=UTF-8";
+                        using (var st = rquest.GetRequestStream())
+                            st.Write(byteArray, 0, byteArray.Length);
+                        var rsponse = (HttpWebResponse)rquest.GetResponse();
+                        var rsponseString = new StreamReader(rsponse.GetResponseStream()).ReadToEnd();
+                        rsponse.Close();
+                        dynamic j1 = JObject.Parse(rsponseString);
```

Comment:
i generated classes like this: public class Rootobject
{
    public int draw { get; set; }
    public int recordsTotal { get; set; }
    public int recordsFiltered { get; set; }
    public Datum[] data { get; set; }
}

public class Datum
{
    public string box_art { get; set; }
    public string title { get; set; }
    public string year { get; set; }
    public string genre { get; set; }
    public string rating { get; set; }
    public string available { get; set; }
    public string cast { get; set; }
    public string director { get; set; }
    public string genre2 { get; set; }
    public string genre3 { get; set; }
    public string slug { get; set; }
}

but .. wont this make the code more vulnerable to changes from allflicks.. i.e. if they added or removed a field etc?